corehq/apps/userreports/README.rst

```diff
@@ -1403,16 +1403,16 @@ Keep in mind that the only variables available for formatting are
 | "%b (%y)" | "Sep (08)"        |
 +-----------+-------------------+
 
-ConditionalAggregationColumn
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ConditionalAggregationRangesColumn
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-**NOTE** This feature is only available to static UCR reports maintained
-by Dimagi developers.
+Conditional aggregation columns allow you to define a series of ranges
+with corresponding names, then group together rows where a specific field's
+value falls within those ranges.
```

Comment:
can you document whether these ranges are inclusive or exclusive?

Reply:
updated (inclusive)

Reply:
Would also be useful to document that conditional aggregation works only with integer columns